indra/newview/llviewerobject.cpp

```diff
@@ -3117,8 +3117,8 @@ struct LLFilenameAndTask
         LL_DEBUGS() << "Destroying LLFilenameAndTask: " << sCount << LL_ENDL;
     }
 private:
-    LLFilenameAndTask(const LLFilenameAndTask& rhs);
-    const LLFilenameAndTask& operator=(const LLFilenameAndTask& rhs) const;
+    LLFilenameAndTask(const LLFilenameAndTask& rhs) = delete;
+    const LLFilenameAndTask& operator=(const LLFilenameAndTask& rhs) = delete;
```

Comment:
VisualStudio was warning about these unimplemented methods.  It looks like someone wanted to restrict the usage of these normally-default methods so I did it correctly using the `delete `keyword and the compiler did not complain.